velox/expression/CastExpr.cpp

```diff
@@ -515,6 +546,23 @@ VectorPtr CastExpr::applyDecimal(
   return castResult;
 }
 
+folly::Function<VectorPtr(
```

Comment:
std::function, but looks like this can be a regular function that returns the result, no?

Reply:
I agree with Masha. We could rename this function and return the result instead.

Reply:
I will change accordingly.